packages/application-generic/src/health/in-memory.health-indicator.ts

```diff
@@ -15,6 +15,10 @@ export class InMemoryProviderServiceHealthIndicator extends HealthIndicator {
     super();
   }
 
+  /**
+   * TODO: Think how to check FeatureFlagsService dependency
+   * without blocking self hosted users
```

Comment:
sorry, but I do not fully understand what you mean... like do you mean to check if the feature flag service is up?

Reply:
maybe better to have a ticket for that, than leaving the todo?